stores/custom-roles/README.md

```diff
@@ -107,58 +105,12 @@ type asset
 ```
 
 > Note: The OpenFGA API accepts a JSON syntax for the authorization model that is different from the DSL shown above
```

Comment:
Do we need to mention this? People don't need to know about the JSON format here, right?